chat-plugins/datasearch.js

```diff
@@ -745,7 +749,7 @@ function runDexsearch(target, cmd, canAll, message) {
 
 			for (let move in alts.moves) {
 				if (!lsetData[mon]) lsetData[mon] = {fastCheck: true, set: {}};
-				if (!TeamValidator('gen7ou').checkLearnset(move, mon, lsetData[mon]) === alts.moves[move]) {
+				if (!TeamValidator(`gen${maxGen === 0 ? 7 : maxGen}ou`).checkLearnset(move, mon, lsetData[mon]) === alts.moves[move]) {
```

Comment:
`` `gen${maxGen || 7}ou` ``

Reply:
Actually, you use this in enough places that you probably just want a `if (!maxGen) maxGen = 7;` somewhere further up in the code.